src/main/java/me/zeroeightsix/kami/module/modules/misc/AutoObsidian.kt

```diff
@@ -196,16 +202,16 @@ object AutoObsidian : Module() {
         /* Updates searching state */
         if (state == State.SEARCHING && searchingState != SearchingState.DONE) {
             searchingState = when {
-                searchingState == SearchingState.PLACING && InventoryUtils.countItem(0, 35, 130) > 0 -> SearchingState.DONE
+                searchingState == SearchingState.PLACING && InventoryUtils.countItemAll(ItemID.ENDERCHEST.id) > 0 -> SearchingState.DONE
                 searchingState == SearchingState.COLLECTING && getDroppedItem(shulkerBoxId, 16.0f) == null -> SearchingState.DONE
                 searchingState == SearchingState.MINING && mc.world.isAirBlock(placingPos) -> {
-                    if (InventoryUtils.countItem(0, 35, 130) > 0) {
+                    if (InventoryUtils.countItemAll(ItemID.ENDERCHEST.id) > 0) {
                         SearchingState.COLLECTING
                     } else { /* In case if the shulker wasn't placed due to server lag */
                         SearchingState.PLACING
                     }
                 }
-                searchingState == SearchingState.OPENING && (InventoryUtils.countItem(0, 35, 130) >= 64 || InventoryUtils.getSlots(0, 35, 0) == null) -> SearchingState.PRE_MINING
+                searchingState == SearchingState.OPENING && (InventoryUtils.countItemAll(ItemID.ENDERCHEST.id) >= 64 || InventoryUtils.getSlots(0, 35, 0) == null) -> SearchingState.PRE_MINING
```

Comment:
You may be wondering why I changed one to countItemAll and the other stayed the same! Well, turns out getSlots() does not nullcheck for slots that may not be there (for example, with an echest we can only go up to 35), while countItem() does. I may submit a PR in the kami repo to add a null check